new-client/src/components/FeatureInfo/FeatureInfoContainer.js

```diff
@@ -4,8 +4,8 @@ import { withStyles } from "@material-ui/core/styles";
 import ArrowLeftIcon from "@material-ui/icons/ArrowLeft";
 import ArrowRightIcon from "@material-ui/icons/ArrowRight";
 import FeaturePropsParsing from "./FeaturePropsParsing";
-import Diagram from "../Diagram";
```

Comment:
We are not supporting diagram anymore?

Reply:
Neither this nor the `Table` shortcode. I asked prior this change and no-one was using it.